atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/OkEvent.java

```diff
@@ -32,15 +33,35 @@
 public abstract class OkEvent implements Event {
     public static final String TYPE = "ok";
 
+    public static final String UNLOCK_SUCCESS = "true";
+    public static final String UNLOCK_FAILURE = "false";
+    public static final String SUCCESS = "Dummy tokens";
+    public static final String FAILURE = "";
+
+    @Override
     public abstract long time();
 
     public abstract int process();
 
     @Nullable
     public abstract String value();
 
+    @JsonProperty("f")
+    public abstract String function();
+
     @Override
     public void accept(EventVisitor visitor) {
         visitor.visit(this);
     }
+
+    public boolean isFailure() {
+        return value() == null
+                || (function().equals(RequestType.UNLOCK) && value().equals(UNLOCK_FAILURE))
+                || value().equals(FAILURE);
+    }
+
+    public boolean isSuccessful() {
+        return value().equals(SUCCESS)
+                || (function().equals(RequestType.UNLOCK) && value().equals(UNLOCK_SUCCESS));
+    }
```

Comment:
These 2 checks seem rather specific to `OkEvent` interpreted _in the context of a lock checker_. I think it would be preferable to have these as static methods in the specific lock checker, or in some utility class that can be used by lock checkers, rather than being a property of the event.